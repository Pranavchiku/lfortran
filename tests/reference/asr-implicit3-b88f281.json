{
    "basename": "asr-implicit3-b88f281",
    "cmd": "lfortran --indent --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/implicit3.f90",
    "infile_hash": "9990e2caf4807e8cd565787f2610168aa6bc7b5cfacc8a4073b1f8d9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-implicit3-b88f281.stdout",
    "stdout_hash": "6c66adce4b76a6157a839d9626cb76f9c1c909376ebf6e6ac2cb6a05",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}