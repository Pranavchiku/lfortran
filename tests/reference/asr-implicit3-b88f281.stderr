warning: Implied save feature
 --> tests/implicit3.f90:8:1
  |
8 | integer :: X(n), Y(m)
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/implicit3.f90:8:1
  |
8 | integer :: X(n), Y(m)
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
